docker-compose.yml

```diff
@@ -0,0 +1,48 @@
+version: '3.8'
+
+services:
+  db:
+    image: postgres:15-alpine
+    ports:
+      - "5432:5432"
+    volumes:
+      - postgres_data:/var/lib/postgresql/data/
+    env_file:
+      - .env
+
+  redis:
+    image: redis:latest
+    ports:
+      - "6379:6379"
+
+  celery:
+    build:
+      context: .
+      dockerfile: Dockerfile.celery
+    volumes:
+      - .:/code
+    depends_on:
+      - db
+      - redis
+    env_file:
+      - .env
+    command: sh -c 'celery -A junction worker -l info -E'
+
+  web:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    volumes:
+      - .:/code
+    ports:
+      - "8888:8888"
+    depends_on:
+      - db
+      - redis
+      - celery
+    env_file:
+      - .env
+    command: sh -c 'python manage.py migrate && python manage.py runsslserver 0.0.0.0:8888'
```

Comment:
wouldn't this be a problem unless DEBUG is always True?

Reply:
I've set DEBUG as FLASE in `.env.sample` and removed the `.env` file as suggested.

<img width="369" alt="image" src="https://user-images.githubusercontent.com/3849885/232708518-66bfc2e8-1b5f-4cc7-b79c-6fd19975ec02.png">

so by default DEBUG is set to FALSE

Reply:
Sorry if I misunderstood your comment earlier.

Reply:
We won't be using the default django server for prod so this is fine. We will use gunicorn in prod.

Reply:
@Sc0rpi0n101 is correct. `runsslserver` has been commented in THIRD_PARTY_APPS. We need to add it to dev.py.sample. Otherwise we will get this error.

![image](https://user-images.githubusercontent.com/10486343/232824223-7e17274e-4d9a-4835-82e4-b552189c8010.png)

Reply:
My bad, in my local I updated `dev.py` and added `runsslserver` there, but didn't update `dev.py.sample`. That would ensure that in the dev settings, we have `runsslserver` added to installed apps. So which is why it works for me. Updating `dev.py.sample`

![image](https://user-images.githubusercontent.com/3849885/232873036-b2553941-88fe-4196-bc4a-4c8a1d9e57cb.png)

Reply:
Pushed the change, let me know if its okay?